### PR TITLE
Make amd usable

### PR DESCRIPTION
**Pull request by Naomiusearch**
Seems like profiling peak memory is broken/really slow for GPTQ, FP16 of LLAMA-3 8b works just fine

### CODE REVIEW

**Comment by Naomiusearch** [comment]
GPTQ seems to work now for now reason(?)

File: docs/pages/installation/installation-rocm.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ Finally, build Aphrodite:
 git clone https://github.com/PygmalionAI/aphrodite-engine.git
 cd aphrodite-engine
 
+chmod +x ./amdpatch.sh
+./amdpatch.sh
```
Comment (AlpinDale) [inline docs/pages/installation/installation-rocm.md:76]: Can we add this to the Dockerfile as well?

Comment (AlpinDale) [reply docs/pages/installation/installation-rocm.md:76]: I may do it in a follow-up PR.

Comment (Naomiusearch) [reply docs/pages/installation/installation-rocm.md:76]: Not sure, cause I don't use docker. Though probably yes

**Pull request merged**